python/emitter.js

```diff
@@ -233,24 +241,33 @@
         },
 
         open({appName, viewportSize}) {
-            let special_branch = '\n    '
-            if ((`${test.config.baselineName}` === 'TestCheckOverflowingRegionByCoordinates_Fluent')
-                || (`${test.config.baselineName}` === 'TestCheckOverflowingRegionByCoordinates_Fluent_Scroll')
-            )
-                special_branch = '\n    eyes.configure.branch_name = \"master_python\"\n    '
+            let new_line = '\n    '
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            let new_line = "\n    "
```